api/src/main/resources/workbench.yaml

```diff
@@ -1221,6 +1221,9 @@ definitions:
       contactEmail:
         description: email address that can be used to contact the user
         type: string
+      firstSignInTime:
```

Comment:
The only other place I see in the API we have a creation date is a string for cluster creation date:

```
      createdDate:
        type: string
        description: The date and time the cluster was created, in ISO-8601 format
```

I generally prefer string format for dates in an API, as I linked to Dan in his cohort PR: https://cloud.google.com/apis/design/naming_convention#time_and_duration. RFC 3339 is a profile of ISO-8601 so this would be roughly compatible with what we're doing for clusters.

Reply:
Yeah, in this case I was actually following the format that we were using for `termsOfServiceCompletionTime` and the other completion times on Profile.